src/cnab.rs

```diff
@@ -14,32 +11,32 @@ use std::str::FromStr;
 ///
 /// The fields here are in canonical order.
 #[derive(Debug, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
```

Comment:
This is not available in v0.8 of Serde, so downgrading means we need to manually rename each field whose name doesn't match the JSON property name.